### PR TITLE
[Relay] Handle pad value coming from Tensor instead of scalar

### PR DESCRIPTION
**Pull request by kparzysz-quic**
The PadCompute function would pass empty index to obtain the pad value. This caused a crash when the pad value was given in a tensor with the following message:
```
Check failed: shape.size() == indices.size() (1 vs. 0) : Tensor dimension mismatch in read ndim = 1, indices.size=0
```

### CODE REVIEW

File: tests/python/unittest/test_pad_value_in_array.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import numpy as np
+
+import tvm
+import tvm.testing
+from tvm import relay
+
+# This used to crash
+
+
+def test_pad_value_in_array():
```
Comment (masahi) [inline tests/python/unittest/test_pad_value_in_array.py:26]: Please move this test to `tests/python/relay/test_op_level2.py` where there are existing tests for `nn.pad`.

Comment (kparzysz-quic) [reply tests/python/unittest/test_pad_value_in_array.py]: Done

**Review by masahi — APPROVED** [review]

**Comment by kparzysz-quic** [comment]
@tvm-bot rerun

**Comment by kparzysz-quic** [comment]
@tvm-bot rerun

**Comment by kparzysz-quic** [comment]
@driazati Do you know why docker/pr-head is marked as "failed", but when I click on "details", there is no indication of a failure there?

**Comment by driazati** [comment]
> @driazati Do you know why docker/pr-head is marked as "failed", but when I click on "details", there is no indication of a failure there?
> 
> Edit: The failure marking is gone now. I'm not sure what happened.

I'm not quite sure myself, the full log: https://ci.tlcpack.ai/job/tvm-docker/job/PR-14735/5/console looks like everything completed successfully but the job says it was cancelled/interrupted (rather than a regular failure).

**Comment by kparzysz-quic** [comment]
> I'm not quite sure myself, the full log: https://ci.tlcpack.ai/job/tvm-docker/job/PR-14735/5/console looks like everything completed successfully but the job says it was cancelled/interrupted (rather than a regular failure).

I think what's happening is that there are several builds running for this PR at the same time, and the latest one to finish notifies the status page.  The wasm build was "failed" for a while, and now it's set to "passed".

**Pull request merged**